src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,25 +8,43 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in this format: <BLOCK, STREET, UNIT, POSTAL_CODE>";
+    public static final String ADDRESS_VALIDATION_REGEX = "(.+,)\\s(.+,)\\s(.+,)\\s(.+)";
 
     public final String value;
     private boolean isPrivate;
 
+    public final Block blockNum;
```

Comment:
As discussed during the tutorial, do change the access modifiers to `private`.

Also, do you think its appropriate to name the objects as `objectNum`?

Reply:
On hindsight, I think it should be renamed to `block` instead. Putting it is blockNum suggests that it is an integer, which is not the case as it can be any string. Will make the necessary changes!